rootfs/api/models/app.py

```diff
@@ -822,3 +833,20 @@ def _build_env_vars(self, build_type, version, image, envs):
         # merge envs on top of default to make envs win
         default_env.update(envs)
         return default_env
+
+    def set_maintenance(self, maintenance):
```

Comment:
Debating if this should live here or in the AppSettings model. What's the benefit of keeping it in `App`?